pkg/agent/controller/networkpolicy/l7engine/reconciler.go

```diff
@@ -76,6 +76,12 @@ outputs:
       types:
         - alert:
             tagged-packets: yes
+        - http:
+            extended: yes
+        - dns:
+            version: 2
```

Comment:
is that necessary? we don't support L7 rules for DNS

Reply:
+1, we shouldn't set this in case it causes extra overhead.

Reply:
Removed.